frontend/simple/views/containers/proposals-form/RuleRemoveMember.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <proposal-template
+    :subTitle="L('Change Rule to Remove Member')"
+    :onSubmit="handleSubmit">
+    I step to change rule of remove member it's on the way
+  </proposal-template>
+  </div>
```

Comment:
Extraneous `</div>`

Reply:
I feel like we should have a linter to catch this kind of thing...

Reply:
Surprising that Vue doesn't crash. 🤔

Reply:
fixed